src/System.Collections.NonGeneric/src/System/Collections/ArrayList.cs

```diff
@@ -64,6 +67,9 @@ public ArrayList()
         // initially empty, but will have room for the given number of elements
         // before any reallocations are required.
         // 
+        /// <summary>Initializes a new instance of the <see cref="T:System.Collections.ArrayList" /> class that is empty and has the specified initial capacity.</summary>
```

Comment:
@Priya91 or @jaredpar could you clarify here?

I believe we'll need to full ID to be able to `f1` the type and open reference docs, but if there's a way to simplify what's shown in the source and still keep that functionality we should definitely do that.  I agree, the fully-qualified ID is too wordy for the source.